extensions/cornerstone/src/utils/measurementServiceMappings/ArrowAnnotate.js

```diff
@@ -130,11 +135,10 @@ function getDisplayText(mappedAnnotations, displaySet) {
     InstanceNumber = instance.InstanceNumber;
   }
 
-  displayText.push(
-    InstanceNumber
-      ? `(S: ${SeriesNumber} I: ${InstanceNumber})`
-      : `(S: ${SeriesNumber})`
-  );
+  const instanceText = InstanceNumber ? ` I: ${InstanceNumber}` : '';
+  const frameText = displaySet.isMultiFrame ? ` F: ${frameNumber}` : '';
+
+  displayText.push(`(S: ${SeriesNumber}${instanceText}${frameText})`);
```

Comment:
This is for displayText so it will not get saved anywhere, but does the user care about the fact that it is multiframe (so is it necessary to have F:...)? I understand that InstanceNumber will be 1 all the time for multiframe.

cc @wayfarer3130

Reply:
This could be the scenario:
When there are multiple annotations on different frames, and we display all those annotations on the right side Measurement Tracking panel. And when the user click on one of the annotations, it should jump the correct frame.
So displaying correct frame number would be great, I believe.